provider/cluster_rosa_classic_resource.go

```diff
@@ -506,6 +506,12 @@ func createClassicClusterObject(ctx context.Context,
 		instanceIamRoles.WorkerRoleARN(state.Sts.InstanceIAMRoles.WorkerRoleARN.Value)
 		sts.InstanceIAMRoles(instanceIamRoles)
 
+		if !state.Sts.OIDCEndpointURL.Unknown && !state.Sts.OIDCEndpointURL.Null && state.Sts.OIDCEndpointURL.Value != "" &&
```

Comment:
I think you should split it - 
it the user set only one of those attributes maybe you should throw an error